README.md

```diff
@@ -30,14 +30,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+.map() returns an array while .forEach does not.
+
 2. What is the difference between a function and a method?
 
+A method is a function that belongs to an object.
+
 3. What is closure?
 
+A closure is created when a function is defined within a function that uses the environment (variables or other functions) that the parent function defined. It is a combination of the child function and the environment where it was defined.
+
```

Comment:
I wouldn't exactly depict is a combination of the two environments, however I see that you understand the idea.